oauth2client/multistore_file.py

```diff
@@ -481,4 +474,7 @@ def _get_storage(self, key):
         Returns:
             A Storage object that can be used to get/set this cred
         """
-        return self._Storage(self, key)
```

Comment:
I haven't had a chance to fully scrub this file. This particular module is really complicated so I want to make sure I'm 100% confident in the Storage design before updating it.